### PR TITLE
Fix cpu and memory stats

### PR DESCRIPTION
**Pull request by rkul**
Fixes https://github.com/prometheus-community/ecs_exporter/issues/40

### CODE REVIEW

**Comment by SuperQ** [comment]
Oh, that makes a lot of sense. I wonder why the CI pipeline is broken. I'll see if I can get it working again.

**Comment by SuperQ** [comment]
Looks like this needs a `go fmt`.

File: ecsmetadata/client.go [hunk 1]
```diff
@@ -94,9 +94,9 @@ type ContainerStats struct {
 	ID       string  `json:"id"`
 	NumProcs float64 `json:"num_procs"`
 
-	CPUStats    dockertypes.CPUStats
-	PreCPUStats dockertypes.CPUStats
-	MemoryStats dockertypes.MemoryStats
+	CPUStats    dockertypes.CPUStats `json:"cpu_stats"`
+	PreCPUStats dockertypes.CPUStats `json:"precpu_stats"`
+	MemoryStats dockertypes.MemoryStats `json:"memory_stats"`
```
Comment (SuperQ) [inline ecsmetadata/client.go:99]:
```suggestion
	CPUStats    dockertypes.CPUStats   `json:"cpu_stats"`
	PreCPUStats dockertypes.CPUStats   `json:"precpu_stats"`
	MemoryStats dockertypes.MemoryStats `json:"memory_stats"`
```

**Comment by rkul** [comment]
Yeah, fixed

**Pull request merged**